<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>query</class>
 <widget class="QWidget" name="query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>714</width>
    <height>416</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>714</width>
    <height>416</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gemi Sorgu</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radHistory">
          <property name="text">
           <string>Kayıt Id'ye göre gemi sorgusu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radType">
          <property name="text">
           <string>Tip Id'ye göre gemi sorgusu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radOwner">
          <property name="text">
           <string>Sahip Id'ye göre gemi sorgusu</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="txtId"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnEnter">
          <property name="text">
           <string>Sorgula</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="table">
      <property name="accessibleName">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
